Xamarin.Forms.Platform.WPF/Properties/AssemblyInfo.cs

```diff
@@ -71,4 +72,6 @@
 
 // Others
 [assembly: Xamarin.Forms.Dependency(typeof(ResourcesProvider))]
-[assembly: Xamarin.Forms.Dependency(typeof(Deserializer))]
\ No newline at end of file
+[assembly: Xamarin.Forms.Dependency(typeof(Deserializer))]
+
+[assembly: AssemblyVersion("2.0.0.0")]
```

Comment:
This changes are related more with https://github.com/xamarin/Xamarin.Forms/issues/11328 just to be able to launch Core Gallery and test everything.